dvc/fs/dvc.py

```diff
@@ -76,6 +76,40 @@ def __init__(
         repo_factory: Optional[RepoFactory] = None,
         **repo_kwargs: Any,
     ) -> None:
+        """DVC + git-tracked files fs.
+
+        Args:
+            path (str, optional): URL or path to a DVC/Git repository.
+                Defaults to a DVC repository in the current working directory.
+                Both HTTP and SSH protocols are supported for remote Git repos
+                (e.g. [user@]server:project.git).
+            rev (str, optional): Any Git revision such as a branch or tag name,
+                a commit hash or a dvc experiment name.
+                Defaults to the default branch in case of remote repositories.
+                In case of a local repository or if the repo is not a Git repo,
+                this option is ignored.
```

Comment:
```suggestion
                Defaults to the default branch in case of Git repositories.
                In case the repo is not a Git repo this option is ignored.
```

@skshetry As far as I have tried, `rev` also works for local Git repositories, right?